packages/kbn-babel-preset/webpack_preset.js

```diff
@@ -20,12 +20,17 @@ module.exports = () => {
         },
       ],
       require('./common_preset'),
-    ],
-    plugins: [
       [
-        require.resolve('babel-plugin-styled-components'),
+        require.resolve('@emotion/babel-preset-css-prop'),
         {
-          fileName: false,
+          labelFormat: '[local]',
+          // importMap: {
+          //   '@elastic/eui': {
+          //     css: {
+          //       canonicalImport: ['@emotion/react', 'css'],
+          //     },
+          //   },
+          // },
```

Comment:
Interesting [option](canvasDropdownFilter) where EUI could re-export `css`, etc from `@emotion/react` and the plugin would handle the transforms.
Untested, unverified.

Reply:
@thompsongl FYI, link doesn't work.

Reply:
link?